### PR TITLE
Configuration test task

### LINKED ISSUE #67: Add a task to test Sentry DSN configuration
In `raven-ruby` you can quickly test your Sentry configuration by running a rake task.

See https://github.com/getsentry/raven-ruby/wiki/Advanced-Configuration#testing-your-configuration

```
$ rake raven:test[https://public:secret@app.getsentry.com/3825]
Client configuration:
-> server: https://app.getsentry.com
-> project_id: 3825
-> public_key: public
-> secret_key: secret

Sending a test event:
-> event ID: 033c343c852b45c2a3add98e425ea4b4

Done!
```

I'd like to propose we have something similar. Maybe a mix task, if appropriate. But I would love to quickly test that my configuration is working from the CLI or some equivalent interface.

### PR DESCRIPTION
**Pull request by mitchellhenke**
closes #67 

@jeregrine

### CODE REVIEW

**Comment by jeregrine** [comment]
Lgtm

File: README.md [hunk 1]
```diff
@@ -78,6 +78,40 @@ config :sentry,
...
+
+Sending test event!
+```
+
+A couple of things to note:
+
+* This won't test your environment configuration. The test CLI forces your
+  configuration to represent itself as if it were running in the production env.
+* If you're running within Rails (or anywhere else that will bootstrap the
+  rake environment), you should be able to omit the DSN argument.
```
Comment (JoshSmith) [inline README.md]: Looks like mention of Rails and rake here missed?

**Comment by JoshSmith** [comment]
This looks great generally :+1:

File: README.md (hunk 1 above)
Comment (mitchellhenke) [inline README.md]: doh, good catch, thank you

**Pull request merged**